.taprc

```diff
@@ -0,0 +1,5 @@
+coverage: true
```

Comment:
isn't coverage true by default?

Reply:
I don't think so, what is true by default is `check-coverage`. let me confirm that

Reply:
Yup! Not necessary indeed: https://node-tap.org/docs/coverage/ - removed

Reply:
okay but can you see the level of additional work that you not looking into this beforehand causes? can you please put more attention to these fairly straightforward details?

Reply:
To be fair, this was a draft for early feedback, but nevertheless - point noted 👍